app/services/sync_provider_from_find.rb

```diff
@@ -128,6 +128,8 @@ def assign_course_attributes_from_find(course, find_course)
     course.course_length = find_course.course_length
     course.recruitment_cycle_year = find_course.recruitment_cycle_year
     course.exposed_in_find = find_course.findable?
+    course.subjects = find_course.subjects
```

Comment:
Could we extract the subject codes here, so we don't have to get them out in `Course#subject_codes`? I'd like the database to be structured as possible.

Reply:
Sure - do you mean _instead of_ storing the whole structure, or _as well as_ ? 

In a previous chat with @tvararu we thought we would probably need to refer to the subjects more in future, so storing the whole structure would give us more options with not much overhead. But if you'd rather only store what we actually need right now, I'm fine with that too.

Reply:
Yes, instead of. I'd like to store what we need, and avoid coupling our code to Find. The sync should ideally translate Find's data into something that's useful for us, so you don't have to understand the Find data model to understand how Apply works.

Reply:
Sure - i've updated the migration to just store `subject_codes` now, and pulled it out to a separate PR( #1882 )